db/migrate/20160318213755_create_gem_download.rb

```diff
@@ -0,0 +1,10 @@
+class CreateGemDownload < ActiveRecord::Migration
+  def change
+    create_table :gem_downloads do |t|
+      t.integer :rubygem_id, null: false
+      t.integer :version_id, null: false
```

Comment:
I'd prefer we not go that route and make it overly generic. Those kinds of text keys are so brittle and we've issues with that kind of thing before.